src/App.php

```diff
@@ -215,53 +213,6 @@ public function processConsole(array $argv): void
 		(new \Friendica\Core\Console($this->container, $argv))->execute();
 	}
 
-	public function processWorker(array $options): void
-	{
-		$this->setupContainerForAddons();
-
-		$this->setupContainerForLogger(LogChannel::WORKER);
```

Comment:
@art4 - with this refactoring, we loose all different logging channels. It's now `console` all the time :-/

One "quick hack" would be to distinguish at `processConsole`, based on the subcommand, like
`if in_array($argv, 'daemon'] { LogChannel::DAEMON }`
but it feels ultra-hacky!

Do you have another idea? The problem is that we setup `DI::` afterwards and create some classes, which then will use the `LoggerInterface`, which makes it hacky as well to switch the channel afterwards. In `Monolog`, it's not even switchable (part of the constructor)

Reply:
I'm afk atm. Solution would be named instances.
https://r.je/dice#example3-6